components/brave_welcome_ui/constants/welcome_types.ts

```diff
@@ -3,11 +3,10 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 export const enum types {
-  IMPORT_NOW_REQUESTED = '@@welcome/IMPORT_NOW_REQUESTED',
+  IMPORT_BROWSER_DATA_REQUESTED = '@@welcome/IMPORT_BROWSER_DATA_REQUESTED',
   GO_TO_TAB_REQUESTED = '@@welcome/GO_TO_TAB_REQUESTED',
   CLOSE_TAB_REQUESTED = '@@welcome/CLOSE_TAB_REQUESTED',
   CHANGE_DEFAULT_SEARCH_PROVIDER = '@@welcome/CHANGE_DEFAULT_SEARCH',
-  IMPORT_DEFAULT_SEARCH_PROVIDERS_STARTED = '@@welcome/IMPORT_DEFAULT_SEARCH_PROVIDERS_STARTED',
   IMPORT_DEFAULT_SEARCH_PROVIDERS_SUCCESS = '@@welcome/IMPORT_DEFAULT_SEARCH_PROVIDERS_SUCCESS',
-  IMPORT_DEFAULT_SEARCH_PROVIDERS_FAILURE = '@@welcome/IMPORT_DEFAULT_SEARCH_PROVIDERS_FAILURE'
+  IMPORT_BROWSER_PROFILES_SUCCESS= '@@welcome/IMPORT_BROWSER_PROFILES_SUCCESS'
```

Comment:
nit: not sure why lint didn't spot this one but there should be a space after `IMPORT_BROWSER_PROFILES_SUCCESS`